docker.Makefile

```diff
@@ -87,6 +88,11 @@ vendor: build_dev_image
 	$(info Vendoring...)
 	docker run --rm $(DEV_IMAGE_NAME) sh -c "make vendor && hack/check-git-diff vendor"
 
+schemas: build_dev_image
```

Comment:
Should the target be called `specification/bindata.go` ? (to map the main `Makefile`)

Reply:
It's a little hard to remember or to understand what it means, no?

Reply:
Then why https://github.com/docker/app/pull/290/files#diff-b67911656ef5d18c4ae36cb6741b7965R98 ?
I mean, it's not that difficult, you're creating/generating the `specification/bindata.go` file

Reply:
(my main concern here, is to use the same target as in `Makefile`)

Reply:
Fixed

Reply:
:thinking: it's still `schemas` :thinking: